### DISCUSSION TITLE
Planetscale requires a credit card to create database and i don't have one

### DISCUSSION THREAD

**Discussion by sumitbishti** [dubinc/dub#533] [General]
I want to contribute in this project but Planetscale requires a credit card to create database. And i don't have one.
How do i contribute now?

**Comment by rahilansari261** [comment]
I think you can use the local database and pass that database url in env file
 DATABASE_URL="mysql://<username>:<password>@localhost:3306/<database>"

Replace <username>, <password>, and <database> with your MySQL credentials

**Comment by steven-tey** [comment]
Good point, thanks for surfacing! We're planning to create test env vars for contributors & local dev purposes soon, stay tuned!

**Reply by versecafe** [reply]
@steven-tey with planetscale [pulling the free tier](https://planetscale.com/blog/planetscale-forever) is there anyway to push up having development vars with DB branches for frequent contributors since running MySQL locally is a large hurdle and makes testing full deployments on Vercel impossible for users without the ability to easily port forward.

**Reply by versecafe** [reply]
On this note it seems to now be impossible to develop locally without the POSTMARK_API_KEY which doesn't have a convenient way to setup a free low use account, every api route using `sendEmail` will cause a crash during build from what I've seen
